views/peeps/index.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% @peeps.each do |peep| %>
+    <li><%= peep.message %> <%= peep.date%> </li> 
+  <% end %>
+</ul>
+
+<a href="/peeps/new">Add Peep</a>
```

Comment:
I like that there is a link to the 'new' page on the index page! Makes it very accessible to the user.